pkg/monitor/context/context.go

```diff
@@ -87,7 +87,7 @@ func (t *traceContext) Commit(msg string) {
 	for _, export := range t.exporters {
 		export(t, duration.Microseconds())
 	}
-	klog.InfoSDepth(1, msg, t.getTagsWith("duration", duration.String())...)
```

Comment:
What is this change for? It is a critical change for controller logging. /cc @leejanee

Reply:
I don't want it to print logs during the execution of CLI. so, I changed its log level.

Reply:
It will change the logging file position without InfoSDepth(1,...). Disable logs in CLI should have other solutions. Not recommend changing the code here.